.travis.yml

```diff
@@ -33,6 +33,7 @@ before_install:
 
 install:
   - composer install --prefer-dist
+  - ./test-js.sh
```

Comment:
why is this task in the install section ?

(for your test you could temporarly comment all the useless lines)